services/core-api/app/api/document_generation/namespace.py

```diff
@@ -1,6 +1,8 @@
 from flask_restplus import Namespace
-from app.api.document_generation.resources.now_document import NoticeOfWorkDocumentResource
+from app.api.document_generation.resources.now_document_resource import NoticeOfWorkDocumentResource
+from app.api.document_generation.resources.explosives_permit_document_resource import ExplosivesPermitDocumentResource
 
 api = Namespace('documents', description='Generate documents for business objects.')
 
-api.add_resource(NoticeOfWorkDocumentResource, '')
```

Comment:
What was this doing prior to this change? The empty string looks a bit suspect, or I'm misunderstanding.

Reply:
This used to be the only resource (for NoW document generation) and was just accessed via just `documents/`, weirdly enough. But now it is `documents/notice-of-work` as we had to add another resource `documents/explosives-permit`.

Reply:
I mentioned above too in a reply to Justin that we should try to generalize the document generation endpoint in a tech debt ticket.

Reply:
Nice, do we have a link to the ticket we can post here for historical purposes?